pixi.toml

```diff
@@ -4,9 +4,12 @@ version = "0.1.0"
 description = "Reader to load data from DICOM files into Apache Arrow"
 authors = ["Marc Garcia <garcia.marc@gmail.com>"]
 channels = ["conda-forge"]
-platforms = ["linux-64"]
+platforms = ["linux-64", "osx-arm64"]  # --> add in "osx-arm64" for MacOS
```

Comment:
I think it's fine to now add these comments, I can see in the diff that you added it, and once committed I don't think it'll be useful.

Reply:
Agreed, was keeping track of things I changed locally while I was testing it. I will remove it.

Reply:
Done